fcrepo-search-impl/src/main/java/org/fcrepo/search/impl/DbSearchIndexImpl.java

```diff
@@ -471,11 +472,7 @@ public void reset() {
     private List<String> toggleForeignKeyChecks(final boolean enable) {
 
         if (isPostgres()) {
-            return List.of(
-                    togglePostgresTriggers(SEARCH_RESOURCE_RDF_TYPE_TABLE, enable),
-                    togglePostgresTriggers(SEARCH_RDF_TYPE_TABLE, enable),
-                    togglePostgresTriggers(SIMPLE_SEARCH_TABLE, enable)
-            );
+            return EMPTY_LIST;
         } else {
             return List.of("SET FOREIGN_KEY_CHECKS = " + (enable ? 1 : 0) + ";");
```

Comment:
Your description says this is now only needed for H2, so could we stop doing this for MySQL and MariaDB too. Maybe repurpose the `isPostgres()` into `isH2()`?

Reply:
@whikloj :  I was wrong - we need the foreign key enable/disable on mysql, h2, and mariadb after all.